src/main/scala/rna16s.scala

```diff
@@ -46,7 +46,10 @@ case object rna16s extends AnyBlastDB {
   /* Here we want to keep sequences which */
   val predicate: (Row, FASTA.Value) => Boolean = { (row, fasta) =>
      /* - are annotated as encoding 16S */
-     (row.select(gene_name) == geneNameFor16S) &&
+     (row.select(gene_name) == geneNameFor16S ||
+      fasta.getV(header).description.toLowerCase.contains("16s") ||
+      fasta.getV(header).description.toLowerCase.contains("small subunit ribosomal rna")
```

Comment:
>We add now in this database
those sequences that have 16S rRNA annotation in the RNACentral table
those that have 16s or 16S in their header description
and those that contain small subunit ribosomal in their header description

Sorry but I need to known perfectly the composition of the database:
You have added these additional sequences to a primary set composed by what sequences? How the sequences were selected?